README.md

```diff
@@ -3,39 +3,49 @@ Stockalike parts pack for Kerbal Space Program
+Now, 500 parts sounds like it would kill your PC. While the mod certainly is large, the memory overhead is likely lower than you would expect. Care has been taken to ensure the mod is efficient, with methods such as enforcing a strict texel density (a technique I have only used since mid-2017, so the older assets might not be as efficient) and texture atlasing to cram several parts onto each texture sheet.
 
-For those familiar with KSP mods, BDB can be considered a stockalike equivalent of FASA, and a US counterpart to Tantares' soviet rockets. 
+For those familiar with KSP mods, BDB can be considered a ReStockalike equivalent of FASA, and a US counterpart to Tantares' soviet rockets. 
 
-Currently, the mod includes (but is not limited to) parts to make:
+Currently, the mod includes (but is not limited to):
 - Mercury spacecraft (Courtesy of Bloeting Aerospace Corp)
 - Gemini spacecraft (Courtesy of Trails Manufacturing Co.)
+- Gemini Ferry
+- Big Gemini 69' (nice)
 - Apollo Block I and II spacecraft, as featured in REAL LIFE!
 - Apollo Block III, III+, IV, and V spacecraft, from Eyes Turned Skywards
 - AARDV "Aardvark" autonomous resupply vehicle, also from Eyes
-- Modular 1.875m station parts for Gemini-era space stations
+- Modular 1.875m and 1.5m station parts for Gemini-era space stations
 - Skylab station parts
 - Spacelab (from Eyes) station parts
+- Parts for the Apollo-Venus mission
 - Vanguard rocket and probe
 - Scout rocket
 - Redstone rocket, with Sergeant clusters and Explorer 1
 - Juno II, IV rockets
-- Diamant rocket (All variants)
 - Thor-Delta rocket (Almost all variants can be made)
-- Delta 2 rocket (Including upper stage and a variety GEM SRBs)
+- Delta 2 and 3 rockets (Including upper stage and a variety GEM SRBs)
+- 4 meter Delta Cryogenic Second Stage (5 meter
 - Atlas rocket (everything from Atlas A up through Atlas V 551)
 - Agena upper stage (All variants)
 - Vega (old Kerolox upper stage for Atlas)
 - Centaur D, G, G', T, V upper stages
 - Titan 1, 2, 3, 4 rockets, with SRMs and Transtage (No IUS yet!)
+- Config switching for LR87's, with alternate kerolox versions and plumes to match!
+- Config switching for H1, RS27, and many more! (All config switching provided courtesy of B9PS)
+- LDC Titan
+- Peacekeeper/Minotaur SRB based launcher
+- Athena SRB based launcher
 - Saturn 1, 1B, V
 - Saturn 1C and Saturn Multibody from Eyes
 - New probe cores
 - New antennas and dishes
 - New science experiments 
-In total, nearly 400 parts!
+- High quality realplumes built with Plume Party FX by Zorg!
+- Full engine ignitor configs by RocketMan
+In total, nearly 500 parts and many features for those parts!
 
 Planned content includes massive Saturn rockets including unflown variants, launchers based on Atlas and Titan that never left the drawing board, and parts based on proposed projects for Gemini and Apollo.
```

Comment:
I believe Atlas and Titan here are referring to SLV3X, SLV3K, CELV, and LDC, all since added.